cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,18 @@
 /// <reference types="cypress" />
-
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
-
-  });
+  beforeEach(() => {
+ cy.visit('https://react-redux.realworld.io')
```

Comment:
Website URL is hard coded. It's better to move it to a configuration or environment file.